docs/upgrading.md

```diff
@@ -0,0 +1,73 @@
+# Upgrading the Agent
+
+This document provides user guidance around agent upgrades and versioning.
+Like every other piece of software included in your project, the agent 
+should be upgraded frequently enough to pick up ongoing enhancements 
+while allowing room for testing. 
+
+## Watching For New Releases
+
+How can you find out about new releases? The best way is to "Watch" this repo
```

Comment:
To find out about new releases, add yourself to the main repository watch list. To do this, click **Watch** on the main repository page. If you don't want notifications for every pull request and issue, choose **Custom** and clear all options except **Release**.